js/bleutrade.js

```diff
@@ -203,4 +211,23 @@ module.exports = class bleutrade extends bittrex {
             throw new ExchangeError (this.id + ' publicGetOrderbook() returneded no result ' + this.json (response));
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
+
+    async fetchMyTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        if (!('orderid' in params)) {
+            throw new ArgumentsRequired (this.id + ' fetchMyTrades required parameter "orderid"');
+        }
+        await this.loadMarkets ();
+        let market = this.markets[symbol];
+        let response = await this.accountGetOrderhistory (params);
+        let trades = this.parseTrades (response['result'], market, since, limit);
+        for (let i = 0; i < trades.length; i++) {
+            let trade = trades[i];
+            trade.id = trade.info.ID;
+            trade.order = params.orderid;
```

Comment:
This won't work with Python, unfortunately:
1. assoc keys should always be used like `trade['order']` and `trade['info']['ID']`.

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

> all associative keys must be single-quoted strings everywhere, `array['good']`, `array.bad`

Also, I think the parsing for `trade['id'] = trade['info']['ID']` should be moved to parseTrade.